web/templates/stream_settings/stream_creation_form.hbs

```diff
@@ -5,32 +5,32 @@
             <div class="alert stream_create_info"></div>
             <div id="stream_creating_indicator"></div>
             <div class="stream-creation-body">
-                <section class="block">
-                    <label for="create_stream_name" class="settings-field-label">
+                <section  class="block create_stream_title_container">
+                    <label for="create_stream_name">
                         {{t "Channel name" }}
                     </label>
                     <input type="text" name="stream_name" id="create_stream_name" class="settings_text_input"
                       placeholder="{{t 'Channel name' }}" value="" autocomplete="off" maxlength="{{ max_stream_name_length }}" />
                     <div id="stream_name_error" class="stream_creation_error"></div>
                 </section>
-                <section class="block">
-                    <label for="create_stream_description" class="settings-field-label">
+                <section  class="block create_stream_description_container">
+                    <label for="create_stream_description">
                         {{t "Channel description" }}
                         {{> ../help_link_widget link="/help/change-the-channel-description" }}
                     </label>
                     <input type="text" name="stream_description" id="create_stream_description" class="settings_text_input"
                       placeholder="{{t 'Channel description' }}" value="" autocomplete="off" maxlength="{{ max_stream_description_length }}" />
                 </section>
 
-                <section class="block" id="make-invite-only">
+                <section  class="block create_stream_types_container" id="make-invite-only">
                     <div class="stream-types">
                         {{> stream_types
                           stream_post_policy=stream_post_policy_values.everyone.code
                           is_stream_edit=false
                           can_remove_subscribers_setting_widget_name="new_stream_can_remove_subscribers_group" }}
                     </div>
                 </section>
-                <section class="block">
+                <section class="block stream_create_add_subscriber_container" style="display: none;">
```

Comment:
It is better to avoid adding inline styles and instead add it in respective CSS files.